lib/cases/case_page.dart

```diff
@@ -26,6 +28,17 @@ class CasePage extends ConsumerWidget {
     print(caseId);
   }
 
+  void updateDocumentWithStatus(DR caseDocRef, STATUS selectedStatus) async {
+    try {
+      // Call the update method with a map containing the field and its new value.
+      await caseDocRef.update({
+        'status': getStatusKey(selectedStatus),
+      });
+    } catch (e) {
+      rethrow; // Rethrow the exception to be caught by the caller.
+    }
+  }
+
```

Comment:
Let's not introduce function that just call other functions without doing anything else.